cmd/bridge/main.go

```diff
@@ -592,6 +592,16 @@ func main() {
 		},
 	}
 
+	go func() {
+		// Listen on port 8080 to be redirected to the console
+		redirectServer := http.NewServeMux()
+		redirectServer.HandleFunc("/", func(res http.ResponseWriter, req *http.Request) {
+			log.Infof("Redirecting from %s to %s", req.Host, listenURL.String())
+			http.Redirect(res, req, listenURL.String()+req.URL.Path, http.StatusMovedPermanently)
```

Comment:
Does this preserve the query parameters?

Reply:
I'm not quite sure I see this logic clearly:

```javascript 
// if custom route set
listenOnDefaultRouteAndRedirectToCustomRoute() // which this console pod is still listening for
// otherwise
listenOnDefaultRouteAndServeAsNormal()
```
I guess console itself doesn't have to know about `custom route` or not, it just needs to know what triggers it to listen at a certain point and act as a redirect server.

Reply:
No this listener will be set all the time. We would need to tell the console, using the console-config.yaml that it needs to listen for a redirect.